app/src/main/java/cm/aptoide/pt/app/AppViewAnalytics.java

```diff
@@ -28,6 +28,7 @@
   public static final String OPEN_APP_VIEW = "OPEN_APP_VIEW";
   public static final String APP_VIEW_INTERACT = "App_View_Interact";
   public static final String CLICK_INSTALL = "Clicked on install button";
+  public static final String DONATIONS_IMPRESSION = "donations_impression";
   private static final String APPLICATION_NAME = "Application Name";
```

Comment:
I asked Ana if there were any standards for the events names and she said that in the past there were no standards, but now we should always write the event name with upper case at the beginning of each word, so this should be Donations_Impression